src/templates/alert_global.html

```diff
@@ -3,7 +3,7 @@
     <h4><% if (messageType.icon) { %><i class="<%=messageType.icon%>"></i> <% } %><%=messageType.title%></h4>
     <ul>
         <% _.each(messages, function (message) { %>
-        <li><%=message%></li>
+        <li><%-message%></li>
```

Comment:
Nope. [`<%-` HTML-escapes the content](https://cdn.rawgit.com/jashkenas/underscore/1.5.2/index.html#template).